src/mlpack/methods/ann/layer/recurrent_attention.hpp

```diff
@@ -139,6 +139,12 @@ class RecurrentAttention
   //! Modify the gradient.
   OutputDataType& Gradient() { return gradient; }
 
+  //! Get the module output size.
+  size_t const& OutSize() const { return outSize; }
+
+  //! Get the number of steps to backpropagate through time.
+  size_t const& Rho() const { return rho; }
+
```

Comment:
Same as as above, i.e., the other constructor parameters can have getters too.

Reply:
Same reason as the one I specified above. ```rnnModule``` and ```actionModule``` are pushed back into the network [here](https://github.com/mlpack/mlpack/blob/ba25b8e139649c8c64e83a0331e1e4388f1088ee/src/mlpack/methods/ann/layer/recurrent_attention_impl.hpp#L54). And then we are returning the network [here](https://github.com/mlpack/mlpack/blob/ba25b8e139649c8c64e83a0331e1e4388f1088ee/src/mlpack/methods/ann/layer/recurrent_attention.hpp#L116).

Reply:
Same as above